src/SDKs/Batch/DataPlane/Azure.Batch/GeneratedProtocol/Models/MultiInstanceSettings.cs

```diff
@@ -35,15 +36,15 @@ public MultiInstanceSettings()
         /// <summary>
         /// Initializes a new instance of the MultiInstanceSettings class.
         /// </summary>
-        /// <param name="numberOfInstances">The number of compute nodes
-        /// required by the task.</param>
         /// <param name="coordinationCommandLine">The command line to run on
         /// all the compute nodes to enable them to coordinate when the primary
         /// runs the main task command.</param>
+        /// <param name="numberOfInstances">The number of compute nodes
+        /// required by the task.</param>
         /// <param name="commonResourceFiles">A list of files that the Batch
         /// service will download before running the coordination command
         /// line.</param>
-        public MultiInstanceSettings(int numberOfInstances, string coordinationCommandLine = default(string), IList<ResourceFile> commonResourceFiles = default(IList<ResourceFile>))
+        public MultiInstanceSettings(string coordinationCommandLine, int? numberOfInstances = default(int?), IList<ResourceFile> commonResourceFiles = default(IList<ResourceFile>))
```

Comment:
@matthchr any reason the order of these parameters flipped? This seems like an easy breaking change to prevent 😄

Reply:
It's in the AutoRest generated code, and the "required-ness" of them changed (spec was actually wrong).